splitio/javascript-client
# Development dependencies update

# JS SDK

## What did you accomplish?
Updated most dev dependencies

## How do we test the changes introduced in this PR?
- Run the `npm run rebuild` command and check there are no failures.
- Run the whole test suite and check there're no errors using `npm run test`.
- Validate TypeScript declarations testing by running `npm run test-ts-decls`.
- Running the compiled builds through a test app would be ideal.

## Extra Notes
- If you already had the repo on your machine, delete the `/node_modules` folder and run `npm install` once more before testing the change.
